game-core/src/main/java/games/strategy/engine/framework/map/download/DownloadMapsWindow.java

```diff
@@ -312,134 +289,77 @@ private JTabbedPane newAvailableInstalledTabbedPanel(
     final JPanel outOfDate =
         outOfDateDownloads.isEmpty()
             ? null
-            : newMapSelectionPanel(selectedMapName, outOfDateDownloads, MapAction.UPDATE);
+            : newMapSelectionPanel(outOfDateDownloads, MapAction.UPDATE);
     // For the UX, always show an available maps tab, even if it is empty
     final JPanel available =
-        newMapSelectionPanel(
-            selectedMapName, mapList.getAvailableExcluding(pendingDownloads), MapAction.INSTALL);
+        newMapSelectionPanel(mapList.getAvailableExcluding(pendingDownloads), MapAction.INSTALL);
 
-    // if there is a map to preselect, show the available map list first
-    if (selectedMapName.isPresent()) {
-      tabbedPane.addTab("Available", available);
-    }
-
-    // otherwise show the updates first
-    if (outOfDate != null) {
-      tabbedPane.addTab("Update", outOfDate);
-    }
-
-    // finally make sure we are always showing the 'available' tab, this condition will be
-    // true if the first 'mapName.isPresent()' is false
-    if (selectedMapName.isEmpty()) {
-      tabbedPane.addTab("Available", available);
-    }
+    tabbedPane.addTab("Available", available);
+    tabbedPane.addTab("Update", outOfDate);
 
     if (!mapList.getInstalled().isEmpty()) {
-      final JPanel installed =
-          newMapSelectionPanel(selectedMapName, mapList.getInstalled(), MapAction.REMOVE);
+      final JPanel installed = newMapSelectionPanel(mapList.getInstalled(), MapAction.REMOVE);
       tabbedPane.addTab("Installed", installed);
     }
     return tabbedPane;
   }
 
   private JPanel newMapSelectionPanel(
-      final Optional<String> selectedMap,
-      final List<DownloadFileDescription> unsortedMaps,
-      final MapAction action) {
-
-    final List<DownloadFileDescription> maps =
-        unsortedMaps.stream()
-            .sorted(Comparator.comparing(lhs -> lhs.getMapName().toUpperCase()))
-            .collect(Collectors.toList());
+      final List<DownloadFileDescription> unsortedMaps, final MapAction action) {
+
+    //    final List<DownloadFileDescription> maps =
+    //        unsortedMaps.stream()
+    //            .sorted(Comparator.comparing(lhs -> lhs.getMapName().toUpperCase()))
+    //            .collect(Collectors.toList());
```

Comment:
Did you mean to keep this comment or should this be removed?

Reply:
Removed, thanks for spotting.
40e8b97fe841ef42bbfe5c3369d3a1b8d4c11bb8